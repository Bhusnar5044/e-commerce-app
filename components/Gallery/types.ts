import { Image } from '@/types';

export interface GalleryProps {
    images: Image[];
}
