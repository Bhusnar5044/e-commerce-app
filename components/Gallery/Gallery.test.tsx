import { render } from '@test-utils';
import { Gallery } from './Gallery';

describe('Gallery', () => {
    it('should render properly', () => {
        const { container } = render(<Gallery images={[]} />);
        expect(container).toMatchSnapshot();
    });
});
