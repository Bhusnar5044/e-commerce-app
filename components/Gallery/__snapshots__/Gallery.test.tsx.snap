// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery should render properly 1`] = `
<div>
  <button
    aria-hidden="true"
    style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px;"
    type="button"
  />
  <div
    class="flex flex-col-reverse"
  >
    <div
      class="mx-auto mt-6 hidden w-full max-w-2xl sm:block lg:max-w-none"
    >
      <div
        aria-orientation="horizontal"
        class="grid grid-cols-4 gap-6"
        role="tablist"
      />
    </div>
    <div
      class="aspect-square w-full"
    />
  </div>
</div>
`;
